lib/Cake/TestSuite/CakeTestCase.php

```diff
@@ -170,40 +141,28 @@ public function tearDown() {
 		unset($this->_configure, $this->_pathRestore);
 	}
 
+	/**
+	 * Sets a static timestamp
+	 *
+	 * @param bool $reset to set new static timestamp.
+	 * @return int timestamp
+	 */
+	public static function time($reset = false) {
+		static $now;
+		if ($reset || !$now) {
+			$now = time();
+		}
+		return $now;
+	}
+
 /**
- * See CakeTestSuiteDispatcher::date()
- *
  * @param string $format format to be used.
  * @return string
  */
 	public static function date($format = 'Y-m-d H:i:s') {
-		return CakeTestSuiteDispatcher::date($format);
+		return date($format, static::time());
 	}
 
-// @codingStandardsIgnoreStart PHPUnit overrides don't match CakePHP
-
-/**
- * Announces the start of a test.
- *
- * @return void
- */
-	protected function assertPreConditions() {
-		parent::assertPreConditions();
-		$this->startTest($this->getName());
-	}
-
-/**
- * Announces the end of a test.
- *
- * @return void
- */
-	protected function assertPostConditions() {
-		parent::assertPostConditions();
-		$this->endTest($this->getName());
-	}
-
```

Comment:
The parent methods of these did absolutely nothing. I guess they did something once upon a time!

Reply:
You are removing CakeTestCase functionality here but removing the 'startTest' and 'endTest' methods being called. I imagine you have done this deliberately, but since you haven't said that anywhere, I am pointing it out.